Include/internal/pycore_sliceobject.h

```diff
@@ -13,6 +13,8 @@ extern "C" {
 
 extern void _PySlice_Fini(PyInterpreterState *);
 
+extern PyObject *
+_PyBuildSlice_Consume2(PyObject *start, PyObject *stop);
```

Comment:
Consume here refers to references of start and stop? Can it be more explicit, maybe ``_PyBuildSlice_ConsumeRefs``?